cmd/retester/retester.go

```diff
@@ -100,6 +98,50 @@ func (b *backoffCache) saveToDisk() (ret error) {
 	return ret
 }
 
+// Config is retester config for all configured repos and orgs.
+// Every level (retester/org/repo) has attributes MaxRetestsForShaAndBase, MaxRetestsForSha, Enabled
+type Config struct {
+	Retester Retester `json:"retester"`
+}
+
+type Retester struct {
```

Comment:
For repo I made it enabled by default when org is configured and enabled. It makes sense for me. Otherwise I had to use `*bool` 
Also when `GetRetesterPolicy` returns false is always with some error:
- "repo is disabled"
- "not configured org"
- "not configured repo and disabled org"

Reply:
The test cases are valid, though.

Yes. I realized now `*bool` is used for branch-protector's config.
Let us go with the pointer.

Reply:
I changed to `*bool` I would say the code is more readable now.
All tests passed.